packages/xo-server-backup-reports/src/index.js

```diff
@@ -395,7 +395,13 @@ class BackupReportsXoPlugin {
         mailReceivers,
         markdown: toMarkdown(markdown),
         success: false,
-        nagiosMarkdown: `[Xen Orchestra] [${log.status}] Backup report for ${jobName} - Error : ${log.result.message}`,
+        nagiosMarkdown: `[Xen Orchestra] [${
+          log.status
+        }] Backup report for ${jobName}${
+          log.result?.message !== undefined
```

Comment:
When? Can you add a specific message for this?

Reply:
There is no result for interrupted Jobs,  IMHO, the message isn't necessary.